File: Praktikum 3. Kondisional/Soal 4/PRAK304-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>

void ejaan_cacah(int bilangan)
{
    if (bilangan == 0)
    {
        printf("nol\n");
    }
    else if (bilangan < 10)
    {
        printf("satuan\n");
    }
    else if (bilangan < 20)
    {
        printf("belasan\n");
    }
    else
    {
        printf("puluhan\n");
    }
}

int main()
{
    int jumlah_bilangan, bilangan;

    printf("Berapa kali input angka? : ");
    scanf("%d", &jumlah_bilangan);

    for (int i = 0; i < jumlah_bilangan; i++)
    {
        printf("Masukkan angka (1-100) : ");
        scanf("%d", &bilangan);
        if (bilangan >= 0 && bilangan < 100)
        {
            ejaan_cacah(bilangan);
        }
        else
        {
            printf("Anda Menginput Melebihi Limit Bilangan\n");
        }
    }

    return 0;
}

File: Praktikum 3. Kondisional/Soal 4/PRAK304-2310817310001-AvantioFierzaPatria.py
def ejaan_cacah(bilangan):
    
    if bilangan == 0:
        print("nol")
    elif bilangan < 10:
        print("satuan")
    elif bilangan < 20:
        print("belasan")
    else:
        print("puluhan")

# Membaca input dari pengguna
jumlah_bilangan = int(input("Berapa kali input angka ? : "))

# Membaca bilangan cacah dan mengonversinya ke ejaan
for _ in range(jumlah_bilangan):
    bilangan = int(input("Masukkan angka (1-100) : "))
    if bilangan >= 0 and bilangan < 100:
         ejaan_cacah(bilangan)
    else:
        print("Anda Menginput Melebihi Limit Bilangan")
